qcodes/instrument_drivers/american_magnetics/AMI430.py

```diff
@@ -123,7 +124,7 @@ def off(self) -> None:
 
     @_Decorators.check_enabled
     def check_state(self) -> bool:
-        return bool(self.ask("PS?").strip())
+        return bool(ast.literal_eval(self.ask("PS?").strip()) )
```

Comment:
but if we know that PS command returns an int, why don't we cast it to int directly making the code simpler?
```suggestion
        return bool(int(self.ask("PS?").strip()))
```

Reply:
@peendebak could you respond? :)

Reply:
@astafan8 Good suggestion. Someone else from my team is looking at more places in the driver where this can occur.